src/tokenize.ts

```diff
@@ -1,27 +1,12 @@
 import { error } from "./error.js";
 import { JsonPosition } from "./position.js";
 import { cannotTokenizeSymbol } from "./tokenizeErrorTypes.js";
-import { JsonToken, ParseSettings } from "./types.js";
-
-export enum JsonTokenTypes {
-  COMMENT = "COMMENT", // // ... \n\r? or /* ... */
-  LEFT_BRACE = "LEFT_BRACE", // {
-  RIGHT_BRACE = "RIGHT_BRACE", // }
-  LEFT_BRACKET = "LEFT_BRACKET", // [
-  RIGHT_BRACKET = "RIGHT_BRACKET", // ]
-  COLON = "COLON", //  :
-  COMMA = "COMMA", // ,
-  STRING = "STRING", //
-  NUMBER = "NUMBER", //
-  TRUE = "TRUE", // true
-  FALSE = "FALSE", // false
-  NULL = "NULL", // null
-  IDENTIFIER = "IDENTIFIER", // identifiers
-}
+import { JsonToken, JsonTokenTypes, ParseSettings } from "./types.js";
 
 interface ParseJsonToken {
   type: JsonTokenTypes;
   value: string;
+  decoded?: string;
```

Comment:
Would it be like this if made more specific with types?
```suggestion
  decoded?: string | null;
```